awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx

```diff
@@ -107,16 +107,22 @@ function LaunchPrompt({ config, resource = {}, onLaunch, onCancel, i18n }) {
   return (
     <Formik
       initialValues={{
-        verbosity: resource.verbosity || 0,
-        inventory: resource.summary_fields?.inventory || null,
-        credentials: resource.summary_fields?.credentials || null,
-        diff_mode: resource.diff_mode || false,
-        extra_vars: resource.extra_vars || '---',
-        job_type: resource.job_type || '',
-        job_tags: resource.job_tags || '',
-        skip_tags: resource.skip_tags || '',
-        scm_branch: resource.scm_branch || '',
-        limit: resource.limit || '',
+        verbosity: config.ask_verbosity_on_launch && (resource.verbosity || 0),
```

Comment:
I'm curious what you were running in to that made you make this change?  Was it something with the preview step or?

Reply:
Precisely. The Preview Step will show the default resource values in  Prompted Values, even if they are not Prompt on Launch values. So this addition prevents those values from being added on load so that they don't show up in the preview step.